src/containers/DropSiteForm.js

```diff
@@ -72,19 +73,22 @@ class DropSiteForm extends React.Component {
         />
         <InputText
           label="Name"
-          value={this.state.name}
-          customOnChange={this.handleFieldChange('contactName')}
+          value={this.state.dropSiteName}
+          customOnChange={this.handleFieldChange('dropSiteName')}
         />
         <InputText
           label="Email or phone number"
-          value={this.state.phone}
-          customOnChange={this.handleFieldChange('contactPhone')}
+          value={this.state.dropSitePhone}
```

Comment:
We might consider introducing some destructuring to reduce all the `this.state`'s

Reply:
merging this into the facility form and will address there!